tmt/steps/discover/fmf.py

```diff
@@ -82,6 +185,31 @@ def wake(self):
             if value:
                 self.data[option] = value
 
+    def check_requires(self, all_requires):
+        """
+        Check requires for possible beakerlib libraries
+
+        Fetch all identified libraries, check their required and
+        recommended packages. Return tuple (requires, recommends)
+        containing lists of regular rpm package names aggregated
+        from all fetched libraries.
+        """
+        requires = []
+        recommends = []
+        for require in all_requires:
+            # Library require
+            try:
+                library = Library(require, parent=self)
+                recommends.extend(library.recommend)
+                # Recursively check for possible dependent libraries
+                require, recommend = self.check_requires(library.require)
```

Comment:
Note: Should we explicitly document in recommend field that libraries are not supported there and user has to use require attribute?

Reply:
There is never enough explicit documentation. Added :)